cmd/manager/exec/manager.go

```diff
@@ -299,6 +299,13 @@ func RunManager() {
 		}
 	}
 
+	// starts the golang profiler
+	go func() {
```

Comment:
This should be optional based on a launch param of the binary I think.

Reply:
Yeah, exposing the profiling endpoint of the Go application is regarded as a security hole by go-sec-scan. It is not recommended in a production env. I will remove it